packages/google_maps_flutter/google_maps_flutter_android/README.md

```diff
@@ -48,7 +48,24 @@ Hybrid Composition, but currently [misses certain map updates][4].
 This mode will likely become the default in future versions if/when the
 missed updates issue can be resolved.
 
+## Map renderer
+
+This plugin supports the option of choosing a preferred [map renderer][5].
+
+Prefer renderer in the application manifest `android/app/src/main/AndroidManifest.xml`:
```

Comment:
This should be a code-based API, not a manifest option. Using the manifest prevents any kind of dynamic selection (server-flag-based rollout, device-specific checks, etc.)

It's fine to have an API that can only be called once, before creating a map, just as the underlying API is.

Reply:
> Using the manifest prevents any kind of dynamic selection (server-flag-based rollout, device-specific checks, etc.)
Good point, my original implementation was actullay API version, as optional parameter of part of MapConfiguration, but as the initialization for renderer works only on first request, changing this flag on MapConfiguration would give users unexcepted behaviour.
For API implementation I suggest creating separate initializeRenderer method under GoogleMapsFlutterPlatform instance. This method would wait response OnMapsSdkInitializedCallback listened and return selected renderer type to to he dart layer giving developer even more control of the plugin behaviour.
I would create separate small helper class to the GoogleMaps plugin to help user to initialize renderer before GoogleMaps widgets are used or any platform views are created.